rfcs/build-system/02-solution-style-ts-configs.md

```diff
@@ -0,0 +1,204 @@
+# RFC: TS solution style config for vNext (per environment configs)
+
+---
+
+_List contributors to the proposal here: @hotell_
+
+## Summary
+
+> **💡 NOTE:**
+>
+> This is proposed changed only for vNext and packages that use new DX ( via ts config path aliases)
+
+We successfully migrated vNext packages [to new DX (1 stage)](https://github.com/microsoft/fluentui/issues/18579), which leverages TS path aliases for improved DX and other benefits...
+
+To continue with our journey to improved DX we need to address leaking global types and other related issues (caused by our current `tsconfig.json` config) for vNext packages.
+
+## Problem statement
+
+> see [Open Issues](#open_issues)
+
+- leaking global types
+- invalid type checking (TS is not following proper `target` and `lib`)
```

Comment:
It might be better to just directly link to the issues here, rather than having them in a separate section